pkg/health/server.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package health
```

Comment:
This should be in `satellite/health` or `satellite/inspector`.

Reply:
done